File: Python/9_PalindromeNumber.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        if x < 0: return False
        reverse, aux = 0, x
        while aux > 0:
            increment = aux % 10
            reverse = reverse * 10 + increment
            aux = aux // 10
        if reverse == x: 
            return True
        return False
        

File: C#/9_PalindromeNumber.cs
public class Solution {
    public bool IsPalindrome(int x) {
        if (x < 0) return false;
        int aux = x, reversed = 0;
        while ( aux > 0 ) {
            int increment = aux % 10;
            reversed = reversed * 10 + increment;
            aux = aux / 10;
        }
        if (reversed == x) return true;
        return false;
    }
}
